ibm/service/vpc/resource_ibm_is_backup_policy.go

```diff
@@ -88,6 +88,45 @@ func ResourceIBMIsBackupPolicy() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"health_reasons": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The reasons for the current health_state (if any).",
+				Elem: &schema.Schema{
```

Comment:
health_reasons have code, more_info and message inside. It should be added in the schema.